src/components/table/SortableTable.tsx

```diff
@@ -114,130 +68,83 @@ function SortableTable(props: SortableTableProps) {
+                                            <TableRow
+                                                hover
+                                                tabIndex={-1}
+                                                key={rowLabel}
                                             >
-                                                {row.name}
-                                            </TableCell>
-                                            <TableCell align="right">
-                                                {row.lastChange}
-                                            </TableCell>
-                                            <TableCell align="right">
-                                                {row.size}
-                                            </TableCell>
-                                            <TableCell align="left">
-                                                {row.owner}
-                                            </TableCell>
-                                            <TableCell align="left">
-                                                {row.description}
-                                            </TableCell>
-                                            <TableCell align="left">
-                                                {row.org}
-                                            </TableCell>
-                                        </TableRow>
-                                    );
-                                })}
+                                                <TableCell
+                                                    component="th"
+                                                    scope="row"
+                                                    padding="none"
+                                                >
+                                                    {row.name}
+                                                </TableCell>
+                                                <TableCell align="right">
+                                                    {row.lastChange}
+                                                </TableCell>
+                                                <TableCell align="right">
+                                                    {row.size}
+                                                </TableCell>
+                                                <TableCell align="left">
+                                                    {row.owner}
+                                                </TableCell>
+                                                <TableCell align="left">
+                                                    {row.description}
+                                                </TableCell>
+                                                <TableCell align="left">
+                                                    {row.org}
+                                                </TableCell>
+                                            </TableRow>
```

Comment:
All the rows are still obviously only work for data that fits this. Leaving in for now since we aren't really using this component yet.